### ISSUE TITLE
More Instructions?

### ISSUE THREAD

**Issue by Yuliang-Zou** [modestyachts/imagenet-vid-robust-example#1]
Hi, thanks for the great work.

Would you mind providing the test code (for example, after getting the 1001-dim logits from an ImageNet pre-trained model, what should I do?) and the test code for YTBB-robust?
